### ISSUE TITLE
4chan.org/banned \\\ ban checking not working when 4chanX is enabled.

### ISSUE THREAD

**Issue by cardgum** [ccd0/4chan-x#3476]
The site gives a "Please unblock challenges.cloudflare.com to proceed." message when attempting to check the current ban status. I am able to do so only after disabling 4chanX. 

The site works, it's just a bit of a nuisance having to disable 4chanX to do so.

This error only started appearing in the last week or so. It wasn't a problem before.

![Screenshot 2024-09-12 at 09-48-05 Verification Required](https://github.com/user-attachments/assets/d427a6fa-3b3a-4899-84d6-7c72475b8af1)

**Comment by vmt86** [comment]
Yeah, I'm having this issue with chrome, haven't tried firefox or any other browser yet
